doc/rtd/development/module_creation.rst

```diff
@@ -28,6 +27,8 @@ Example
 
     This will likely take multiple lines.
     """
+    
```

Comment:
There are some trailing whitespaces that Sphinx does not like, (`tox -e doc` to reproduce it):

```suggestion

```